test/automation/src/sql/notebook.ts

```diff
@@ -50,18 +50,36 @@ export class Notebook {
 		await this.code.waitForElement('.notebook-cell.active');
 	}
 
+	async waitForActiveCellGone(): Promise<void> {
+		await this.code.waitForElementGone('.notebook-cell.active');
+	}
+
 	async runActiveCell(): Promise<void> {
 		await this.code.dispatchKeybinding('F5');
 	}
 
+	async exitActiveCell(): Promise<void> {
+		await this.code.dispatchKeybinding('escape');
+		await this.code.dispatchKeybinding('escape');
+	}
+
 	async runAllCells(): Promise<void> {
 		await this.code.dispatchKeybinding('ctrl+shift+F5');
 	}
 
 	// Cell Actions
 
-	async waitForTypeInEditor(text: string) {
-		const editor = '.notebook-cell.active .monaco-editor';
+	async getActiveCell(): Promise<IElement> {
+		const activeCellSelector = '.notebook-cell.active';
+		return this.code.waitForElement(activeCellSelector);
+	}
+
+	async waitForActiveCell(id: string): Promise<void> {
+		await this.code.waitForElement(`.notebook-cell.active[id="${id}"]`);
```

Comment:
This is basically doing the same thing as `getActiveCell` but just not returning the element. I'd just combine the two into a single `getActiveCell(id?: string)` and then have the selector either be the one with or without the ID based on whether an id was passed in (like you do for waitForTypeInEditor)